.pre-commit-config.yaml

```diff
@@ -202,6 +202,12 @@ repos:
           - --convention=pep257
           - --add-ignore=D100,D102,D104,D105,D107,D200,D205,D400,D401
         exclude: ^tests/.*\.py$|^scripts/.*\.py$|^dev|^backport_packages|^kubernetes_tests
+  - repo: https://gitlab.com/pycqa/flake8
```

Comment:
Is not it the case that flake requires to have the full Airflow installed as a local virtualenv locally to be able to run all the Type checks? I think you need to have all the dependencies installed in the same versions that are in the CI  and all the dependencies have to be installed in the right versions..

Reply:
The problem running it in breeze solves is that when we do not have the common environment, people start reporting various errors that are coming from different versions of the environments they have, versions of python, versions of dependencies installed and potentially other dependencies. 

Any idea how we can handle this case?

Reply:
Yup flake8 is definitely tied to the Python version that is installed, that might be a good reason to keep it in breeze.